portfolio/src/main/webapp/index.html

```diff
@@ -93,21 +92,40 @@ <h2>Courses Taken</h2>
       <div class="info-row info-row1">
 		<div class="container">
 			<div class="row">
-				<center><div class="about">
-					<div class="clearfix"></div>
+				<div class="about">
+                    <center>
```

Comment:
this tag is obsolete: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center